intelmq/bots/BOTS

```diff
@@ -789,6 +789,18 @@
                 "port": "<port>"
             }
         },
+        "UDP": {
+            "description": "UDP is a simple UDP bot responsible to send events to a udp port (e.g.: syslog daemon).",
+            "module": "intelmq.bots.outputs.udp.output",
+            "parameters": {
+                "field_delimiter": "|",
+                "format": "<json/delimited> read README.md",
+                "header": "<header text>",
+                "ip": "<ip>",
+                "port": "<port>",
+                "raw_field": "<keep/drop>"
```

Comment:
how about changing this field to "keep_raw_field": true/false?
This way we can use the built in true and false boolean values of JSON.